web_widget_one2many_product_picker/README.rst

```diff
@@ -0,0 +1,136 @@
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_widget_one2many_product_picker
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_widget_one2many_product_picker
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Adds the 'one2many_product_picker' friendly mobile widget to create one2many lines linked with product.template records.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Create or edit a new view and use the new widget called 'one2many_product_picker'.
+
+Widget options:
+~~~~~~~~~~~~~~~
+
+* product_per_page > Integer -> Used to control the auto-load behaviour (scroll-pagination)
+* groups > Array of dictionaries -> Declare the groups
+* field_map > Dictionary:
+
+    * product -> The field that represent the product_id
+    * name -> The field that represent a name
+    * product_uom -> The field that represent a product_uom
+    * product_uom_qty -> The field that represent a product_uom_qty
+    * price_unit -> The field that represent a price_unit
+    * parent_id -> The field that represent the main record
+
+
+Group dictionary definition:
+
+* name -> The group name
+* domain -> Forced domain to use
+* order -> The order
+
+    * name -> The field name to order
+    * asc -> Flag to use 'asc' order
+
+
+Example:
+
+```t-options="{'groups': [{'name': 'Cheap', 'domain': [('list_price', '<', 10.0)], 'field_map': { 'product': 'my_product_id' }}]}"```
+
+
+Also you need define the view fields. The view must be an 'kanban' view.
+This is an example that uses the 'sale.order.line' fields:
+
+.. code:: xml
+
+    <field
+        name="order_line"
+        attrs="{'readonly': [('state', 'in', ('done','cancel'))]}"
+        nolabel="1"
+        widget="one2many_product_picker"
+        options="{'groups': [{'name': 'Desk', 'domain': [('name', 'ilike', '%desk%')], 'order': {'name': 'id', 'asc': true}}, {'name': 'Chairs', 'domain': [('name', 'ilike', '%chair%')]}]}"
+    >
+        <kanban>
+            <field name="name"/>
+            <field name="product_id" />
+            <field name="product_uom_qty" />
+            <field name="product_uom"/>
+            <field name="price_unit"/>
+            <field name="order_id"/>
+        </kanban>
+    </field>
+
+
+** In this example we don't use 'field_map' option because the default match with the sale.order.line field names.
```

Comment:
it could be great to add a "more complex / complete usage", for exemple, (I guess) the implementation for the ``purchase.order.line``. Don't you think ?